src/plugins/intel_cpu/src/nodes/mkldnn_gather_node.cpp

```diff
@@ -113,12 +113,19 @@ void MKLDNNGatherNode::initSupportedPrimitiveDescriptors() {
     if (!supportedPrimitiveDescriptors.empty())
         return;
 
+    impl_desc_type type = impl_desc_type::ref_any;
+    if ((x64::mayiuse(x64::avx512_common) || x64::mayiuse(x64::avx2)) &&
+            (!isDynamicNode() && (x64::mayiuse(x64::avx512_common) && afterAxisSize < x64::cpu_isa_traits<x64::avx512_common>::vlen / dataTypeSize ||
+                    x64::mayiuse(x64::avx2) && afterAxisSize < x64::cpu_isa_traits<x64::avx2>::vlen / dataTypeSize) ||
+            isDynamicNode() && isDataShapeStat && isAxisInputConst && afterAxisSize == 1)) {
+        type = jit;
```

Comment:
Is it possible to set `jit_avx512` or `jit_avx2` depending on the target?

Reply:
Done